### DISCUSSION TITLE
Enable root SSH question

### DISCUSSION THREAD

**Discussion by Tylerjet** [tteck/Proxmox#1432] [Q&A] [answered]
Hey tteck,

Wanted to sanity check on this before making a bug report, it seems that for some LXC scripts setting yes for "Root SSH" does not set ```PermitRootLogin yes```. Is that expected? as according to the discussion here in #385 setting yes should do that process automatically?

For example just tested Bazarr and Node-Red. both of them deny password ssh login until going into sshd_config and modifying again.

But then just tested Jackett and after it was created, password login did work without any intervention

a root password is being set during creation.

**Comment by tteck** [comment, marked as answer]
The configuration mentioned is established within the `install.func` script, which is shared among all LXC scripts. It is reasonable to expect that if it functions correctly for one script, it should function correctly for all scripts.

https://github.com/tteck/Proxmox/blob/db8d22f721ceed662517c4a2766c9ae09988bc19/misc/install.func#L123

**Reply by Tylerjet** [reply]
Of course after making a post it seems to be fine now for both Bazarr and node-red 😆 so I'll just chalk it up to something odd on my end haha.
